provider/azure/internal/imageutils/images.go

```diff
@@ -132,7 +200,7 @@ func ubuntuSKU(ctx context.ProviderCallContext, series, stream, location string,
 		versions = append(versions, version)
 	}
 	if len(versions) == 0 {
-		return "", "", errors.NotFoundf("Ubuntu SKUs for %s stream", stream)
+		return "", "", errors.NotFoundf("Old style Ubuntu SKUs for %s stream", stream)
```

Comment:
s/Old style/legacy

We should use "legacy" instead of "OldStyle" for method names etc too.

Existing code, but let's include the series in the error also